Makefile

```diff
@@ -403,6 +403,10 @@ svcat:
 	$(MAKE) svcat-for-$(CLIENT_PLATFORM)
 	cp $(BINDIR)/svcat/$(TAG_VERSION)/$(CLIENT_PLATFORM)/$(ARCH)/svcat$(FILE_EXT) $(BINDIR)/svcat/
 
+svcat-install: svcat
+	cp $(BINDIR)/svcat/svcat /usr/local/bin/
+	$(BINDIR)/svcat/svcat install plugin
```

Comment:
`/usr/local/bin` is on `PATH`?

Reply:
doh! That is my own customization. Any ideas for how I can make this usable for other people?

Reply:
gobin kind of makes sense? I'm not sure there's a 100% solution here. e.g. windows